images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -181,6 +181,11 @@ function Get-RubyVersion {
     return "Ruby $rubyVersion"
 }
 
+function Get-RubyGemsVersion {
+    $rubyGemsVersion = Run-Command "gem --version"
+    return "RubyGems $rubyGemsVersion"
+}
+
```

Comment:
Should this function be in Linux software report? Currently, it is in macOS

Reply:
Looks like I love macOS more than Linux, fixed